### PR TITLE
Fix wasm_vm_test.

### PR DESCRIPTION
**Pull request by jplevyak**
Signed-off-by: John Plevyak <jplevyak@gmail.com>

### CODE REVIEW

**Comment by PiotrSikora** [comment]
It looks that we are not running those tests on CircleCI.

Could you add `//test/extensions/common/wasm/...` target to `.circleci/config.yml`?

File: .circleci/config.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
    release:
      executor: ubuntu-build
      environment:
-       BAZEL_TEST_TARGETS: "//test/extensions/access_loggers/wasm/... //test/extensions/filters/http/wasm/... //test/extensions/wasm/..."
+       BAZEL_TEST_TARGETS: "//test/extensions/access_loggers/wasm/... //test/extensions/filters/http/wasm/... //test/extensions/wasm/... //test/extensions/commmon/wasm/..."
```
Comment (PiotrSikora) [inline .circleci/config.yml]: Type, `s/commmon/common/` (here and everywhere else).

Comment (jplevyak) [reply .circleci/config.yml]: done

**Comment by PiotrSikora** [comment]
`examples/wasm/envoy.yaml` and `source/extensions/common/wasm/wasm_vm.h` should be also updated.

**Pull request merged**